fastlane_core/spec/cert_checker_spec.rb

```diff
@@ -112,10 +112,12 @@ class ProcessStatusMock
           `ls`
 
           keychain = "keychain with spaces.keychain"
-          cmd = %r{curl -f -o (([A-Z]\:)?\/.+) https://www\.apple\.com/certificateauthority/AppleWWDRCAG6\.cer && security import \1 -k #{Regexp.escape(keychain.shellescape)}}
+          check_cmd = %r{curl -f -o (([A-Z]\:)?\/.+) https://www\.apple\.com/certificateauthority/AppleWWDRCAG6\.cer && security verify-cert -c \1}
```

Comment:
Looking at the implementation I think those commands are now called separately (two separate calls to `Open3.capture3`) rather than a single `curl … && security …` combined command?

Reply:
The test now "only" checks that the `FASTLANE_WWDR_USE_HTTP1_AND_RETRIES` arguments gets passed, like the description indicates.